Assets/Scripts/CloneOnDrag.cs

```diff
@@ -5,12 +5,21 @@ namespace MoveToCode {
     public class CloneOnDrag : MonoBehaviour {
         ManipulationHandler manipulationHandler;
         Vector3 startingPosition;
+        Quaternion rotation;
+        Quaternion startingRotation;
         GameObject clone;
         GameObject codeBlockType;
         bool blockStillInMenu;
 
         private void Start() {
             startingPosition = transform.position;
+            startingRotation = transform.rotation;
+        }
+
+        private void Update()
+        {
+            startingPosition = transform.position;
```

Comment:
I'm slightly confused as to why you reassign this over and over? Is this something you want to do `OnDisable()` or the like?